File: Basic/Java/src/samples/SdsType.java
package samples;


public class SdsType {

    private String Id = "";
    private String Name = "";
    private String Description = "";
    private SdsTypeCode SdsTypeCode;
    private SdsTypeProperty[] Properties = new SdsTypeProperty[0];

    public String getId() {
        return Id;
    }

    public void setId(String id) {
        this.Id = id;
    }

    public String getName() {
        return Name;
    }

    public void setName(String name) {
        this.Name = name;
    }

    public String getDescription() {
        return Description;
    }

    public void setDescription(String description) {
        this.Description = description;
    }

    public SdsTypeCode getSdsTypeCode() {
        return SdsTypeCode;
    }

    public void setSdsTypeCode(SdsTypeCode sdsTypeCode) {
        this.SdsTypeCode = sdsTypeCode;
    }

    public SdsTypeProperty[] getProperties() {
        return Properties;
    }

    public void setProperties(SdsTypeProperty[] properties) {
        this.Properties = properties;
    }
}

File: ocs_samples/basic_samples/SDS/DotNet/SdsRestApiCore/SdsRestApiCore/SdsType.cs
// <copyright file="SdsType.cs" company="OSIsoft, LLC">
//
// </copyright>

using System;
using System.Collections.Generic;
using System.Text;

namespace SdsRestApiCore
{
    public class SdsType
    {
        public string Id
        {
            get;
            set;
        }

        public string Name
        {
            get;
            set;
        }

        public string Description
        {
            get;
            set;
        }

        public SdsTypeCode SdsTypeCode
        {
            get;
            set;
        }

        public IList<SdsTypeProperty> Properties
        {
            get;
            set;
        }
    }
}
